File: src/clojure/gdx/scenes/scene2d/Stage.java
package clojure.gdx.scenes.scene2d;

import com.badlogic.gdx.graphics.g2d.Batch;
import com.badlogic.gdx.utils.viewport.Viewport;

public class Stage extends com.badlogic.gdx.scenes.scene2d.Stage  {

  public Object ctx;

	public Stage (Viewport viewport, Batch batch, Object ctx) {
		super(viewport, batch);
    this.ctx = ctx;
	}

}

File: src/clojure/gdx/scenes/scene2d/stage.clj
(ns clojure.gdx.scenes.scene2d.stage
  (:import (com.badlogic.gdx.scenes.scene2d Stage)))

(defn act! [stage]
  (Stage/.act stage))

(defn draw! [stage]
  (Stage/.draw stage))

(defn add! [^Stage stage actor]
  (.addActor stage actor))

(defn clear! [^Stage stage]
  (.clear stage))

(defn root [^Stage stage]
  (.getRoot stage))

(defn hit [stage [x y]]
  (Stage/.hit stage x y true))
